src/adafruit_blinka/microcontroller/bcm283x/pin.py

```diff
@@ -116,7 +118,12 @@ def value(self, val=None):
     (1, TXD, RXD),
 )
 
-i2cPorts = (
-    (1, SCL, SDA),
-)
-
+# Set correct i2c bus - original Pi used 0, subsequent models use 1:
+if detector.board.RASPBERRY_PI_B_REV1:
```

Comment:
It feels a bit weird to be doing this in the pin definition file for the bcm283x, but I wasn't quite sure how else to go about it.